core/testutils/src/main/java/com/mongodb/stitch/core/testutils/sync/SyncIntTestProxy.kt

```diff
@@ -1767,8 +1839,18 @@ class SyncIntTestProxy(private val syncTestRunner: SyncIntTestRunner) {
         BsonDocument("_id", documentId)
 
     private val failingConflictHandler = ConflictHandler { _: BsonValue, event1: ChangeEvent<Document>, event2: ChangeEvent<Document> ->
-        println("conflict local event: " + event1.fullDocument.toJson())
-        println("conflict remote event: " + event2.fullDocument.toJson())
+        val localEventDescription = when (event1.operationType == ChangeEvent.OperationType.DELETE) {
+            true -> "delete"
+            false -> event1.fullDocument.toJson()
+        }
+
+        val remoteEventDescription = when (event2.operationType == ChangeEvent.OperationType.DELETE) {
+            true -> "delete"
+            false -> event2.fullDocument.toJson()
+        }
+
+        println("conflict local event: $localEventDescription")
```

Comment:
delete?

Reply:
these were already here, just not null-safe, I'm keeping them because they're useful for debugging